urgent-care/src/main/java/gov/va/api/health/urgentcare/service/api/UrgentCareService.java

```diff
@@ -4,11 +4,23 @@
 import gov.va.api.health.r4.api.MetadataApi;
 import io.swagger.v3.oas.annotations.ExternalDocumentation;
 import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
 import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.security.OAuthFlow;
+import io.swagger.v3.oas.annotations.security.OAuthFlows;
+import io.swagger.v3.oas.annotations.security.OAuthScope;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
 import io.swagger.v3.oas.annotations.servers.Server;
 import javax.ws.rs.Path;
 
 @OpenAPIDefinition(
+  security =
+      @SecurityRequirement(
+        name = "OauthFlow",
+        scopes = {"patient/CoverageEligibilityResponse", "metadata"}
```

Comment:
I'm not entirely familiar with this, but based on how it's set up in `data-query`, this should be `patient/CoverageEligibilityResponse.read`, and the `metadata` scope can be removed.